althea_kernel_interface/src/hardware_info.rs

```diff
@@ -437,6 +441,57 @@ fn get_wifi_station_info(dev: &str) -> Vec<WifiStationData> {
     extract_wifi_station_data(&res)
 }
 
+pub fn get_radio_ssid(radio: &str) -> Option<String> {
+    let radio = radio.replace("wlan", "radio");
+    let path = format!("wireless.default_{radio}.ssid");
+    match KI.get_uci_var(&path) {
+        Ok(a) => Some(a),
+        Err(e) => {
+            error!("Unable to get radio ssid for radio: {} with {:?}", radio, e);
+            None
+        }
+    }
+}
+
+pub fn get_radio_enabled(radio: &str) -> Option<bool> {
+    let radio = radio.replace("wlan", "radio");
+    let path = format!("wireless.{radio}.channel");
+    match KI.get_uci_var(&path) {
+        Ok(a) => Some(!a.contains('0')),
+        Err(e) => {
+            error!(
+                "Unable to get radio channel for radio: {} with {:?}",
+                radio, e
+            );
+            None
+        }
+    }
+}
+
+pub fn get_radio_channel(radio: &str) -> Option<u16> {
+    let radio = radio.replace("wlan", "radio");
```

Comment:
this doesn't specify it's expected input or check that it's not getting a junk string input. At least comment what's the expected input format is since it seems to be doing a transformation on it.